api/check.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Check struct {
+	ID                 uuid.UUID `json:"id"`
+	LastTransitionTime time.Time `json:"last_transition_time"`
+	Status             string    `json:"status"`
+}
+
+// AsMap returns a map[string]any representation of the Check object
+// with some additional fields.
+func (t Check) AsMap() map[string]any {
+	m := make(map[string]any)
+	b, _ := json.Marshal(t)
+	_ = json.Unmarshal(b, &m)
+
+	transitionDuration := time.Since(t.LastTransitionTime)
+	m["transition_duration_sec"] = int(transitionDuration.Seconds())
```

Comment:
Can we change this to age = time.Now() - last_transition? 

and then the expression becomes `check.status == 'healthy' && age > duration("15m")` 

Can we also add the duration and `created` age in